x-pack/plugins/ml/server/models/job_audit_messages/job_audit_messages.ts

```diff
@@ -400,9 +407,72 @@ export function jobAuditMessagesProvider(
     return (Object.keys(LEVEL) as LevelName[])[Object.values(LEVEL).indexOf(level)];
   }
 
+  /**
+   * Retrieve list of errors per job.
+   * @param jobIds
+   */
+  async function getJobsErrors(jobIds: string[], earliestMs?: number): Promise<JobsErrorsResponse> {
```

Comment:
This function could take the error level as a parameter, possibly defaulting to `MESSAGE_LEVEL.ERROR`, to make it more reusable.

Reply:
yeah, I was thinking about it but not sure about the use case, i.e. if we ever want to retrieve warnings or info messages

Reply:
yeah, I can't see a use case at the moment, it would just make it potentially more reusable for no extra cost, especially if it had a default message level set to error.
If not i think the function should be renamed to `getJobsErrorMessages`, to conform to the general naming convention in the file.